packages/tokens/.swcrc

```diff
@@ -23,6 +23,14 @@
         "useSpread": true
       }
     },
+    "experimental": {
+      "plugins": [
+        [
+          "@swc/plugin-transform-imports",
+          { "^(.*?)(\\.js)?$": { "skipDefaultConversion": true, "transform": "{{matches.[1]}}.js" } }
```

Comment:
the plugin doesn't fork for exports ATM https://github.com/swc-project/plugins/issues/246#issuecomment-1959256102 


**side questions:** where did you get the api for this ? I wasn't able to find anything besides some folks mentioning some stuff within GH issues

Reply:
I got the API from the GH issues :)